### DISCUSSION TITLE
How to find the files for new platform

### DISCUSSION THREAD

**Discussion by QianpengLi577** [The-OpenROAD-Project/OpenROAD-flow-scripts#421] [Q&A] [answered]
I dont know how to find files for new platform. I have found some open source PDK such as [FreePDK3](https://github.com/ncsu-eda/FreePDK3), but this PDK dont have *.def , *.gds , *.lib file. Maybe I need some softwares such as Synopsys Custom Compiler, IC Validator, Star-RC, HSPICE to generate these files?

**Comment by vijayank88** [comment, marked as answer]
@QianpengLi577 
If its an OpenSource directly contact respective Repo owner for the files. Or you've to follow the steps given in the repo to generate the lef/lib.

**Reply by QianpengLi577** [reply]
Thanks for you reply !
